src/core/Akka.Cluster/Routing/ClusterRoutingConfig.cs

```diff
@@ -120,75 +143,133 @@ protected static int GetMaxTotalNrOfInstances(Config config)
+            get
+            {
+                return Local.SupervisorStrategy;
+            }
+        }
+
+        public override RouterConfig WithFallback(RouterConfig routerConfig)
+        {
+            var otherClusterRouterPool = routerConfig as ClusterRouterPool;
+
+            if (otherClusterRouterPool != null && otherClusterRouterPool.Local is ClusterRouterPool)
+            {
+                throw new ConfigurationException("ClusterRouterPool is not allowed to wrap a ClusterRouterPool");
+            }
+
+            if (otherClusterRouterPool != null)
+            {
+                return Copy(Local.WithFallback(otherClusterRouterPool.Local).AsInstanceOf<Pool>());
+            }
+
+            return Copy(Local.WithFallback(routerConfig).AsInstanceOf<Pool>());
         }
 
         public override Router CreateRouter(ActorSystem system)
         {
             return Local.CreateRouter(system);
         }
 
-        public override Group WithDispatcher(string dispatcher)
+        public override string RouterDispatcher
         {
-            return new ClusterRouterGroup(Local.WithDispatcher(dispatcher), Settings);
+            get
+            {
+                return Local.RouterDispatcher;
+            }
         }
 
-        internal override RouterActor CreateRouterActor()
+        public override bool StopRouterWhenAllRouteesRemoved
         {
-            return new ClusterRouterGroupActor(Settings);
+            get
+            {
+                return false;
+            }
         }
 
-        public override bool IsManagementMessage(object message)
+        public override Props RoutingLogicController(RoutingLogic routingLogic)
         {
-            return message is ClusterEvent.IClusterDomainEvent || message is ClusterEvent.CurrentClusterState || base.IsManagementMessage(message);
+            return Local.RoutingLogicController(routingLogic);
         }
 
-        public override ISurrogate ToSurrogate(ActorSystem system)
+        public override bool IsManagementMessage(object message)
         {
-            throw new NotImplementedException();
+            return message is ClusterEvent.IClusterDomainEvent
+                || message is ClusterEvent.CurrentClusterState
+                || base.IsManagementMessage(message);
         }
 
-        public override RouterConfig WithFallback(RouterConfig routerConfig)
+        public override ISurrogate ToSurrogate(ActorSystem system)
         {
-            var localFallback = routerConfig as ClusterRouterGroup;
-            if (localFallback != null && (localFallback.Local is ClusterRouterGroup)) throw new ConfigurationException("ClusterRouterGroup is not allowed to wrap a ClusterRouterGroup");
-            if (localFallback != null) return Copy(Local.WithFallback(localFallback.Local).AsInstanceOf<Group>());
-            return Copy(Local.WithFallback(routerConfig).AsInstanceOf<Group>());
+            throw new NotImplementedException();
```

Comment:
So an update on this: `ISurrogate` is used for serialization of built-in Akka.NET primitives. Not needed for clustered routers because they're not meant to be remotely deployed onto other nodes.